ooni/ui/web/client/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  </head>
+  <body ng-app="app" ng-strict-di ng-cloak>
+
+    <app>
+      Loading...
```

Comment:
:+1: for adding the web app to allow people to use it from git builds, I think this makes easier for people to test ooni-probe. With this change in place the usefulnees of an event handler is reduced, because, if it does not work, it's because it's broken and not because of user-that-did-not-downloaded-the-latest-gui error.

Besides, how do I fetch the latest version of the gui for testing purposes?